.gitignore

```diff
@@ -0,0 +1 @@
+node_moduules/
```

Comment:
This file should contain the `node_modules` and the `test.md`. Kindly check your spelling :bookmark_tabs: Ensure that these two are not pushed into your repository